common/libs/wallet/base.js

```diff
@@ -1,6 +1,6 @@
 // @flow
-import { stripHex } from 'libs/values';
-import type { RawTransaction } from 'libs/transaction';
+import { stripHex } from '../values';
+import type { RawTransaction } from '../transaction';
```

Comment:
@wbobeirne Yup, I'm aware of this plugin, but I'm not aware on how to ensure that it runs in combination with the npm command: `babel-node common/derivation-checker.js --presets es2015,stage-0,flow`